frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-utilization.tsx

```diff
@@ -59,6 +77,10 @@ export const VMUtilizationCard: React.FC = () => {
     [vmName, namespace, launcherPodName],
   );
 
+  const adjustDuration = React.useMemo(() => {
```

Comment:
It would be easier to read if we use useCallback here `useCallback((start: number) => adjustDurationForVMI...)`

Reply:
Done. And reducing the dependency to the `creationTimestamp` only.